### DISCUSSION TITLE
Possibility of a docker container for a standalone instance?

### DISCUSSION THREAD

**Discussion by tomc128** [JLyne/LiveAtlas#614] [General]
Hey! So with itzg/minecraft-server being such a popular image for dockerising Minecraft servers, it would be great if LiveAtlas could be easily dockerised too. So far, I'm experimenting with creating my own image, however I think it would be great if a custom image for LiveAtlas could be created to more easily set one up.

**Comment by exkc** [comment]
yeah just throw LiveAtlas and your fav web server like nginx Apache lighthttpd  uhttpd Caddy in docker
